### PR TITLE
1864 health check alignment

### PR DESCRIPTION
**Pull request by CodeReaper**

### CODE REVIEW

File: domains/oidc-mock/API/configuration.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: eo-oidc-mock
-version: 1.0.1
+version: 1.0.3
```
Comment (ymersod) [inline domains/oidc-mock/API/configuration.yaml:2]: bump fra 1.0.1 -> 1.0.3

File: domains/certificates/Query.API/API.IntegrationTests/RegistryConnectorHealthTests.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    {
+        this.factory = factory;
+        this.factory.RabbitMqOptions = rabbitMqContainer.Options;
+    }
+
+    [Fact]
+    public async Task Health_IsCalled_ReturnsOk()
+    {
+        using var client = factory.CreateClient();
+        using var healthResponse = await client.GetAsync("health");
```
Comment (ymersod) [inline domains/certificates/Query.API/API.IntegrationTests/RegistryConnectorHealthTests.cs:26]: /health

**Review by ymersod — COMMENTED** [review]
- Certificates - Are program.cs healthcheck URI "/health"?
- transfer/Transfer.API - Are program.cs healthcheck URI "/health"?

**Comment by CodeReaper** [comment]
> * Certificates - Are program.cs healthcheck URI "/health"?
> * transfer/Transfer.API - Are program.cs healthcheck URI "/health"?

Yes they are. :)

**Pull request merged**